[0m12:12:47.618888 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000211CE425F70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000211D05C7290>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000211D05C6210>]}


============================== 12:12:47.623887 | 4c7b1fbc-4ed3-426a-a4ec-c28f3be89ed5 ==============================
[0m12:12:47.623887 [info ] [MainThread]: Running with dbt=1.8.6
[0m12:12:47.624887 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\user\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init dbt_project', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m12:12:47.651894 [debug] [MainThread]: Starter project path: C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\include\starter_project
[0m12:12:47.668415 [info ] [MainThread]: 
Your new dbt project "dbt_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m12:12:47.669414 [info ] [MainThread]: Setting up your profile.
[0m12:12:59.618690 [error] [MainThread]: Encountered an error:

[0m12:12:59.643150 [error] [MainThread]: Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\cli\requires.py", line 138, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\cli\requires.py", line 101, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\cli\main.py", line 470, in init
    results = task.run()
              ^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\task\init.py", line 347, in run
    self.setup_profile(profile_name)
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\task\init.py", line 262, in setup_profile
    adapter = self.ask_for_adapter_choice()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\task\init.py", line 243, in ask_for_adapter_choice
    numeric_choice = click.prompt(prompt_msg, type=click.INT)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\click\termui.py", line 164, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\click\termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m12:12:59.647583 [debug] [MainThread]: Command `dbt init` failed at 12:12:59.647583 after 12.18 seconds
[0m12:12:59.647583 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000211D0CE4890>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000211D0AA7170>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000211D01E9E20>]}
[0m12:12:59.648595 [debug] [MainThread]: Flushing usage events
[0m12:13:37.213697 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000256C7782BD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000256C557A180>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000256C7547560>]}


============================== 12:13:37.216696 | 54692368-2818-439c-9637-fed60d002a6f ==============================
[0m12:13:37.216696 [info ] [MainThread]: Running with dbt=1.8.6
[0m12:13:37.217687 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\user\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init dbt_project', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m12:13:37.230763 [info ] [MainThread]: A project called dbt_project already exists here.
[0m12:13:37.231966 [debug] [MainThread]: Command `dbt init` succeeded at 12:13:37.231966 after 0.12 seconds
[0m12:13:37.233063 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000256C708B050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000256C7507A40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000256C7E9F080>]}
[0m12:13:37.233063 [debug] [MainThread]: Flushing usage events
[0m12:14:12.907915 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000215336DEF30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000215335A75F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000215306BCE00>]}


============================== 12:14:12.910914 | 8a5adf2d-b623-461e-ad28-05e350e6f8fb ==============================
[0m12:14:12.910914 [info ] [MainThread]: Running with dbt=1.8.6
[0m12:14:12.911918 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\user\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt init dbt_project', 'send_anonymous_usage_stats': 'True'}
[0m12:14:12.926033 [debug] [MainThread]: Starter project path: C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\include\starter_project
[0m12:14:12.940680 [info ] [MainThread]: 
Your new dbt project "dbt_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m12:14:12.941682 [info ] [MainThread]: Setting up your profile.
[0m12:17:03.797500 [info ] [MainThread]: Profile dbt_project written to C:\Users\user\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m12:17:03.799422 [debug] [MainThread]: Command `dbt init` succeeded at 12:17:03.799422 after 170.98 seconds
[0m12:17:03.800423 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021532AD8740>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000215439D4C50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021533694D40>]}
[0m12:17:03.801423 [debug] [MainThread]: Flushing usage events
[0m12:55:05.660594 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000239EF94AF90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000239F13D5640>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000239F2755E80>]}


============================== 12:55:05.660594 | b93d3e83-e56f-43f7-a2b0-443356bf0941 ==============================
[0m12:55:05.660594 [info ] [MainThread]: Running with dbt=1.8.6
[0m12:55:05.660594 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': 'C:\\Users\\user\\.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt run', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m12:55:05.660594 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\user\OneDrive - Sails Software Solutions Pvt Ltd\Desktop\learn_dbt\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m12:55:05.660594 [debug] [MainThread]: Command `dbt run` failed at 12:55:05.660594 after 0.16 seconds
[0m12:55:05.660594 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000239F359FC80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000239EF94AF90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000239F3697D70>]}
[0m12:55:05.660594 [debug] [MainThread]: Flushing usage events
[0m11:08:25.640301 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023560DF9010>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023560DF8FE0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023560DFB980>]}


============================== 11:08:25.653332 | 12875dfb-3cb3-4a3e-b047-30eff13ace97 ==============================
[0m11:08:25.653332 [info ] [MainThread]: Running with dbt=1.8.6
[0m11:08:25.653332 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\user\\.dbt', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt docs generate', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m11:08:25.655854 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\user\OneDrive - Sails Software Solutions Pvt Ltd\Desktop\learn_dbt\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m11:08:25.655854 [debug] [MainThread]: Command `dbt docs generate` failed at 11:08:25.655854 after 0.10 seconds
[0m11:08:25.655854 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002355FF17830>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023560E37D40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023560E35850>]}
[0m11:08:25.655854 [debug] [MainThread]: Flushing usage events
[0m12:11:26.560469 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002000336CB00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000200023C95B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002000255BB30>]}


============================== 12:11:26.560469 | d23c95b8-1b78-4b11-b183-c7e5f5458a8c ==============================
[0m12:11:26.560469 [info ] [MainThread]: Running with dbt=1.8.6
[0m12:11:26.560469 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\user\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt init buybox_Data', 'send_anonymous_usage_stats': 'True'}
[0m12:11:26.576172 [debug] [MainThread]: Starter project path: C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\include\starter_project
[0m12:11:26.620801 [info ] [MainThread]: 
Your new dbt project "buybox_Data" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m12:11:26.620801 [info ] [MainThread]: Setting up your profile.
[0m12:14:48.823427 [error] [MainThread]: Encountered an error:
while scanning a simple key
  in "C:\Users\user\.dbt\profiles.yml", line 14, column 1
could not find expected ':'
  in "C:\Users\user\.dbt\profiles.yml", line 14, column 5
[0m12:14:48.859551 [error] [MainThread]: Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\cli\requires.py", line 138, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\cli\requires.py", line 101, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\cli\main.py", line 470, in init
    results = task.run()
              ^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\task\init.py", line 347, in run
    self.setup_profile(profile_name)
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\task\init.py", line 263, in setup_profile
    self.create_profile_from_target(adapter, profile_name=profile_name)
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\task\init.py", line 180, in create_profile_from_target
    self.create_profile_from_profile_template(profile_template, profile_name)
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\task\init.py", line 167, in create_profile_from_profile_template
    self.write_profile(profile, profile_name)
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\task\init.py", line 153, in write_profile
    profiles = yaml.safe_load(f) or {}
               ^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\yaml\__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\yaml\__init__.py", line 81, in load
    return loader.get_single_data()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\yaml\constructor.py", line 49, in get_single_data
    node = self.get_single_node()
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\yaml\composer.py", line 36, in get_single_node
    document = self.compose_document()
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\yaml\composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\yaml\composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\yaml\composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\yaml\parser.py", line 98, in check_event
    self.current_event = self.state()
                         ^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\yaml\parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\yaml\scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\yaml\scanner.py", line 173, in fetch_more_tokens
    return self.fetch_stream_end()
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\yaml\scanner.py", line 377, in fetch_stream_end
    self.remove_possible_simple_key()
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\yaml\scanner.py", line 318, in remove_possible_simple_key
    raise ScannerError("while scanning a simple key", key.mark,
yaml.scanner.ScannerError: while scanning a simple key
  in "C:\Users\user\.dbt\profiles.yml", line 14, column 1
could not find expected ':'
  in "C:\Users\user\.dbt\profiles.yml", line 14, column 5

[0m12:14:48.859551 [debug] [MainThread]: Command `dbt init` failed at 12:14:48.859551 after 202.49 seconds
[0m12:14:48.859551 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020003082A20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000200141B8200>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020014396EA0>]}
[0m12:14:48.859551 [debug] [MainThread]: Flushing usage events
[0m12:15:24.027189 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017522E39E50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017525070230>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000175250702C0>]}


============================== 12:15:24.027189 | 868096b2-1ccd-497b-935a-b439be63342f ==============================
[0m12:15:24.027189 [info ] [MainThread]: Running with dbt=1.8.6
[0m12:15:24.027189 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\user\\.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt init buybox_Data', 'send_anonymous_usage_stats': 'True'}
[0m12:15:24.049279 [info ] [MainThread]: A project called buybox_Data already exists here.
[0m12:15:24.051667 [debug] [MainThread]: Command `dbt init` succeeded at 12:15:24.051667 after 0.21 seconds
[0m12:15:24.051667 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001752594AF00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001752577A990>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017524C4E9F0>]}
[0m12:15:24.051667 [debug] [MainThread]: Flushing usage events
[0m12:17:53.420833 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BE5D9CC050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BE5B7F62A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BE581C9340>]}


============================== 12:17:53.420833 | f6138181-cead-4a4d-a059-79e46f0b73ff ==============================
[0m12:17:53.420833 [info ] [MainThread]: Running with dbt=1.8.6
[0m12:17:53.420833 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\user\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'invocation_command': 'dbt init buybox_data', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m12:17:53.436523 [debug] [MainThread]: Starter project path: C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\include\starter_project
[0m12:17:53.485247 [info ] [MainThread]: 
Your new dbt project "buybox_data" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m12:17:53.485247 [info ] [MainThread]: Setting up your profile.
[0m12:19:26.430689 [info ] [MainThread]: Profile buybox_data written to C:\Users\user\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m12:19:26.444612 [debug] [MainThread]: Command `dbt init` succeeded at 12:19:26.444612 after 93.12 seconds
[0m12:19:26.445618 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BE6F08B3E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BE6E8D6E40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BE5D9CC050>]}
[0m12:19:26.447088 [debug] [MainThread]: Flushing usage events
[0m12:26:40.014592 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000231866EA000>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023188920050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000231889207A0>]}


============================== 12:26:40.014592 | f199b257-68cf-48e5-bf2e-c768de331a5d ==============================
[0m12:26:40.014592 [info ] [MainThread]: Running with dbt=1.8.6
[0m12:26:40.014592 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\user\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt init example', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m12:26:40.035357 [debug] [MainThread]: Starter project path: C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\include\starter_project
[0m12:26:40.063395 [info ] [MainThread]: 
Your new dbt project "example" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m12:26:40.063395 [info ] [MainThread]: Setting up your profile.
[0m12:28:04.779026 [info ] [MainThread]: Profile example written to C:\Users\user\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m12:28:04.781380 [debug] [MainThread]: Command `dbt init` succeeded at 12:28:04.781380 after 84.92 seconds
[0m12:28:04.781380 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000231866EA000>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002318608AF60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023199F9ED80>]}
[0m12:28:04.782505 [debug] [MainThread]: Flushing usage events
